autoload/codi.vim

```diff
@@ -305,7 +327,7 @@ function! s:codi_hide()
   let codi_bufnr = s:get_codi('bufnr')
   if s:get_opt('autoclose') && codi_bufnr && !s:updating
     " Remember width for when we respawn
-    call s:let_codi('width', winwidth(bufwinnr(codi_bufnr)))
+    call s:let_codi('width', s:pane_width())
```

Comment:
The intent behind this line was to save the current window width for a buffer, allowing persistence of width when hiding/showing Codi on a per-buffer basis. We should revert this change.